.cirrus.yml

```diff
@@ -9,7 +9,7 @@ task:
       - JULIA_VERSION: nightly
   allow_failures: $JULIA_VERSION == 'nightly'
   install_script:
-    - sh -c "$(fetch https://raw.githubusercontent.com/ararslan/CirrusCI.jl/master/bin/install.sh -o -)"
+    - sh -c "$(https://raw.githubusercontent.com/ararslan/CirrusCI.jl/master/bin/install.sh -o -)"
```

Comment:
```suggestion
    - sh -c "$(fetch https://raw.githubusercontent.com/ararslan/CirrusCI.jl/master/bin/install.sh -o -)"
```

Reply:
Looks like you caught that 🙂

Reply:
Thanks and sorry for the noise!

Reply:
No problem at all. I'm just happy that someone is actually using CirrusCI.jl 😄